Overview.src.html

```diff
@@ -1363,6 +1419,16 @@ <h4 id=terminology-headers>Headers</h4>
   <p>Let <var>connection</var> be the result of establishing an HTTP connection to
   <var>origin</var>. <span data-anolis-ref>HTTP</span> <span data-anolis-ref>TLS</span>
 
+  <p>If <var>credentials</var> is true and if the user agent supports
+  <span title=concept-token-binding>Token Binding</span>, propose the use of Token Binding for the
+  TLS connection by adding a Token Binding Negotiation extension in the TLS Client Hello as
+  described in
+  <a href="https://tools.ietf.org/html/draft-ietf-tokbind-negotiation#section-2">section 2</a>
+  of the Token Binding Negotiation spec.
+
+  <p class="note no-backref">If Token Binding Negotiation succeeds, the user agent can use
+  <span title=concept-token-binding>Token Binding</span> with the server.
```

Comment:
I'd suggest adding an explicit property to the connection that describes the process in a little more detail. That is, during negotiation, both parties establish a `token_binding_version` and `key_parameters_list`, etc. @annevk will likely have opinions here about what level of detail is appropriate, but I'd suggest abstracting all that into something like `<dfn for="connection">toking binding metadata</dfn>` which is either `null` or an opaque object (as far as Fetch is concerned) that you can pass into the `Sec-Token-Binding` header generation algorithm.